Add cache support for the capabilites request #734 (#1078)
IBM/FHIR: docs/src/pages/guides/FHIRServerUsersGuide.md

|`fhirServer/core/checkReferenceTypes`|boolean|Indicates whether reference type checking is performed by the server during parsing / deserialization.|
|`fhirServer/core/serverRegistryResourceProviderEnabled`|boolean|Indicates whether the server registry resource provider should be used by the FHIR registry component to access definitional resources through the persistence layer.|
|`fhirServer/core/conditionalDeleteMaxNumber`|integer|The max number of matches supported in conditional delete. |

> |`fhirServer/core/capabilityStatementCacheTimeout`|integer|The max time a CapabilityStatement is cached for the metadata endpoint. |

Reviewer: The description needs to specify the units.
```suggestion
|`fhirServer/core/capabilityStatementCacheTimeout`|integer|The number of minutes that a tenant's CapabilityStatement is cached for the metadata endpoint. |
```
